src/Microsoft.Health.Dicom.Core/Features/Operations/DicomDurableFunctionsHttpClient.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Messages.Operations;
+using Microsoft.Health.Dicom.Core.Models.Operations;
+using Microsoft.Net.Http.Headers;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Dicom.Core.Features.Operations
+{
+    internal class DicomDurableFunctionsHttpClient : IDicomOperationsClient
+    {
+        private readonly HttpClient _client;
+        private readonly OperationsConfiguration _config;
+        private static readonly JsonSerializerSettings JsonSettings = new JsonSerializerSettings
+        {
+            DateTimeZoneHandling = DateTimeZoneHandling.Utc,
+        };
+
+        public DicomDurableFunctionsHttpClient(HttpClient client, IOptions<OperationsConfiguration> config)
+        {
+            EnsureArg.IsNotNull(client, nameof(client));
+            EnsureArg.IsNotNull(config?.Value, nameof(config));
+
+            client.BaseAddress = config.Value.BaseAddress;
+            client.DefaultRequestHeaders.Add(HeaderNames.Accept, MediaTypeNames.Application.Json);
+
+            _client = client;
+            _config = config.Value;
+        }
+
+        public async Task<OperationStatusResponse> GetStatusAsync(string id, CancellationToken cancellationToken = default)
```

Comment:
Well, not necessarily! Only if the method is invoked through the interface is it unnecessary. If I invoked it on the concrete type, the default would still be necessary.

Reply:
As I remember, there was discussion regarding where to put this default value. One point is that the implementation class could have different default value than the interface, to avoid such confusion, we only put default value at interface.

Reply:
That's an interesting point. As least for `CancellationToken`, there is really only 1 default value we could have. But for other types -- yeah the default could change. Good point